mpc_controller:
  ros__parameters:
    goal_file: "/home/phak/ros2/src/mpc_planner/goal/goal.txt"
    T: 0.1
    N: 20.0
    linear_mpc:
      # 状态权重: x, y, yaw
      q_weights: [1.0, 1.0, 1.0]
      # 控制权重: v, omega
      r_weights: [2.0, 0.2]
      # 速度限制: v_min, omega_min
      u_lower: [-0.0, -2.0]
      # 速度限制: v_max, omega_max
      u_upper: [0.5, 2.0]